### DISCUSSION TITLE
Recursive mode

### DISCUSSION THREAD

**Discussion by Jordinthecloud** [KBlixt/subcleaner#46] [Q&A] [answered]
Why doesn't this work? I got tons of SRTs in volume1

![image](https://github.com/KBlixt/subcleaner/assets/51406064/eb45171d-3e7d-4b64-8d99-6c166f6f2bd4)

The script is supposed to run in recursive mode.

**Comment by KBlixt** [comment]
I haven't tried running my script against network shares that isn't mounted like in linux, is it possible to mount the network share to the machine and point subcleaner to that directory?

**Comment by Jordinthecloud** [comment]
They’re already mounted to my Windows machine. I tried what you asked but then the same error as in the screenshot appears.

What I tried: python .\subcleaner.py -r U:/Tv shows —dry-run

**Reply by KBlixt** [reply]
If you have spaces in the path I'd recommend putting the path between quotation mark:

python .\subcleaner.py -r "U:/Tv shows" —dry-run

**Comment by Jordinthecloud** [comment]
![image](https://github.com/KBlixt/subcleaner/assets/51406064/8f228c37-d400-4a37-81f8-030136ac3e16)

**Reply by KBlixt** [reply]
Hmm 🤔

What if you just used "Y:/"

**Reply by albino1** [reply]
Maybe the problem is your slash is backwards.  You have `Y:/` and it should be `Y:\` for Windows.

**Comment by Jordinthecloud** [comment]
Same error

**Reply by KBlixt** [reply]
Strange, would you mind showing what it looks like in the Y drive?

**Comment by Jordinthecloud** [comment]
Sure:

![image](https://github.com/KBlixt/subcleaner/assets/51406064/f63014f1-928f-4c4c-9978-970c0ba00247)

**Reply by KBlixt** [reply]
Are you able to run it against a specific subtitle file? Or does that fail as well?

**Comment by Jordinthecloud** [comment]
When I copy an srt to my Windows machine and run the script it works like a charm! But running it against an srt directly on my Synology it fails.

Edit; are you on Discord? I feel that works a bit nicer..

**Reply by KBlixt** [reply]
Yeah, I can't speak until tomorrow though. Pm me your handlename and I'll invite you tomorrow 👍

**Reply by Jordinthecloud** [reply]
lakitna

**Reply by Jordinthecloud** [reply]
Waiting for your invite..

**Reply by KBlixt** [reply]
Starting up computer now.

**Comment by KBlixt** [comment, marked as answer]
This issue is hopefully resolved now. Probably an issue with how windows deals with network storage that detach and reattach while windows is in sleep mode.
